accord-core/src/main/java/accord/local/Command.java

```diff
@@ -963,7 +970,7 @@ public boolean isEqualOrFuller(Command command)
 
     public static class PreAccepted extends AbstractCommand
     {
-        private final Timestamp executeAt;
+        private final @Nullable Timestamp executeAt;
```

Comment:
I don't think this should ever be null? If we're `AcceptedInvalidated` (or, logically, `PreAcceptedInvalidated` that doesn't currently exist as a reified state but is a side-state of `NotDefined`) then we shouldn't be a `PreAccepted` object.

Reply:
I wish I documented why I made this change.  There was a NPE I think in `equals` as it didn't handle `null`... let me see if I can find the code path.